examples/basic-env-vars.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chaostoolkit-env
+  namespace: chaostoolkit-run
+data:
+  FIRST: "Jane"
+  SECOND: "Doe"
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chaostoolkit-experiment
+  namespace: chaostoolkit-run
+data:
+  experiment.json: |
+    {
+      "version": "1.0.0",
+      "title": "Hello world!",
+      "description": "Say hello world.",
+      "configuration": {
+          "first": {
+              "type": "env",
+              "key": "FIRST"
+          },
+          "second": {
+              "type": "env",
+              "key": "SECOND"
+          }
+
+      },
+      "method": [
+        {
+          "type": "action",
+          "name": "say-hello",
+          "provider": {
+            "type": "process",
+            "path": "echo",
+            "arguments": "hello ${first} ${second}"
+          }
+        }
+      ]
+    }
+---
+apiVersion: chaostoolkit.org/v1
+kind: ChaosToolkitExperiment
+metadata:
+  name: my-chaos-exp
+  namespace: chaostoolkit-crd
+spec:
+  namespace: chaostoolkit-run
+  pod:
```

Comment:
I would suggest to remove the pod block with chaosArgs, as it's not what your example is intended to show.

Reply:
from lines 53 to 57

Reply:
fixed